examples/wines.get.json

```diff
@@ -347,11 +347,57 @@
                                     "locale": "it",
                                     "value": "Expovina"
                                 }
-                            ]
+                            ],
+                            "sort_index": 10,
+                            "in_search": true
                         },
                         "year": 2016,
                         "wine_year": 2013,
-                        "valuation": "Gold"
+                        "valuation": {
+                            "type": "diploma",
+                            "text": [
+                                {
+                                    "locale": "de",
+                                    "value": "Grosses Golddiplom"
+                                },
+                                {
+                                    "locale": "fr",
+                                    "value": "Grand diplôme d'Or"
+                                },
+                                {
+                                    "locale": "it",
+                                    "value": "Grande Diploma d'oro"
+                                }
+                            ]
+                        },
+                        "text": [
```

Comment:
This text is the combination of the expert and the valuation. Currently, I don't see an use case for the Wineshop.